src/main/java/teammates/common/datatransfer/FeedbackSessionAttributes.java

```diff
@@ -316,6 +316,24 @@ public boolean isClosingWithinTimeLimit(int hours) {
                && differenceBetweenDeadlineAndNow >= hours - 1
                && differenceBetweenDeadlineAndNow < hours;
     }
+    
+    /**
+     * returns true if the session is closed within the past hour of calling this function
```

Comment:
I believe it's already handled; there's another necessary condition "nowMillis > deadlineMillis" which precisely means "now" must be after "deadline".